2021-02-24 15:54:50,083 INFO    MainThread:15928 [wandb_setup.py:_flush():70] setting env: {}
2021-02-24 15:54:50,083 INFO    MainThread:15928 [wandb_setup.py:_flush():70] multiprocessing start_methods=spawn
2021-02-24 15:54:50,083 INFO    MainThread:15928 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-24 15:54:50,083 INFO    MainThread:15928 [wandb_init.py:_log_setup():318] Logging user logs to C:\Users\Juhyeon\Desktop\repository\Proxy-Anchor-CVPR2020\code\wandb\run-20210224_155450-2r21o7yu\logs\debug.log
2021-02-24 15:54:50,084 INFO    MainThread:15928 [wandb_init.py:_log_setup():319] Logging internal logs to C:\Users\Juhyeon\Desktop\repository\Proxy-Anchor-CVPR2020\code\wandb\run-20210224_155450-2r21o7yu\logs\debug-internal.log
2021-02-24 15:55:03,059 INFO    MainThread:15928 [wandb_run.py:_console_start():1399] atexit reg
2021-02-24 15:55:03,060 INFO    MainThread:15928 [wandb_run.py:_redirect():1262] redirect: SettingsConsole.WRAP
2021-02-24 15:55:03,061 INFO    MainThread:15928 [wandb_run.py:_redirect():1297] Wrapping output streams.
2021-02-24 15:55:03,061 INFO    MainThread:15928 [wandb_run.py:_redirect():1313] Redirects installed.
2021-02-24 15:55:03,062 INFO    MainThread:15928 [wandb_run.py:_config_callback():663] config_cb None None {'LOG_DIR': '../logs', 'dataset': 'cars', 'sz_embedding': 512, 'sz_batch': 128, 'nb_epochs': 60, 'gpu_id': 0, 'nb_workers': 0, 'model': 'googlenet', 'loss': 'Triplet', 'optimizer': 'adamw', 'lr': 0.0001, 'weight_decay': 0.0001, 'lr_decay_step': 20, 'lr_decay_gamma': 0.5, 'alpha': 32, 'mrg': 0.1, 'IPC': None, 'warm': 1, 'bn_freeze': 1, 'l2_norm': 1, 'remark': ''}
2021-02-24 15:55:42,419 INFO    MainThread:15928 [wandb_run.py:_atexit_cleanup():1369] got exitcode: 1
2021-02-24 15:55:42,419 INFO    MainThread:15928 [wandb_run.py:_restore():1341] restore
2021-02-24 15:55:42,631 INFO    MainThread:15928 [wandb_run.py:_wait_for_finish():1492] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1274
  total_bytes: 1274
}

2021-02-24 15:55:44,633 INFO    MainThread:15928 [wandb_run.py:_wait_for_finish():1492] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7328
  total_bytes: 7328
}

2021-02-24 15:55:45,717 INFO    MainThread:15928 [wandb_run.py:_show_files():1714] logging synced files
2021-02-24 15:55:45,719 INFO    MainThread:15928 [internal.py:handle_exit():156] Internal process exited
